zcash_client_backend/src/constants/regtest.rs

```diff
@@ -0,0 +1,38 @@
+/// The testnet coin type for ZEC, as defined by [SLIP 44].
+///
+/// [SLIP 44]: https://github.com/satoshilabs/slips/blob/master/slip-0044.md
+pub const COIN_TYPE: u32 = 1;
+
+/// The HRP for a Bech32-encoded testnet [`ExtendedSpendingKey`].
+///
+/// Defined in [ZIP 32].
+///
+/// [`ExtendedSpendingKey`]: zcash_primitives::zip32::ExtendedSpendingKey
+/// [ZIP 32]: https://github.com/zcash/zips/blob/master/zip-0032.rst
+pub const HRP_SAPLING_EXTENDED_SPENDING_KEY: &str = "secret-extended-key-regtest";
+
+/// The HRP for a Bech32-encoded testnet [`ExtendedFullViewingKey`].
+///
+/// Defined in [ZIP 32].
+///
+/// [`ExtendedFullViewingKey`]: zcash_primitives::zip32::ExtendedFullViewingKey
+/// [ZIP 32]: https://github.com/zcash/zips/blob/master/zip-0032.rst
+pub const HRP_SAPLING_EXTENDED_FULL_VIEWING_KEY: &str = "zxviewregtestsapling";
+
+/// The HRP for a Bech32-encoded testnet [`PaymentAddress`].
+///
+/// Defined in section 5.6.4 of the [Zcash Protocol Specification].
```

Comment:
This is defined in the `zcashd` codebase ([here](https://github.com/zcash/zcash/blob/128d863fb8be39ee294fda397c1ce3ba3b889cb2/src/chainparams.cpp#L493)), not in the protocol specification.

Reply:
Done